environment/wansview-q6.uenv.txt

```diff
@@ -3,18 +3,16 @@ day_night_min=500
 disable_eth=true
 enable_updates=true
 gpio_button=14
-gpio_default=
-gpio_ir850=49
+gpio_ir940=49
```

Comment:
if you can see the IR LED with a naked eye, it's 850nm, not 940

Reply:
It is not visible with the naked eye.